profiling/src/io.rs

```diff
@@ -723,11 +649,7 @@ impl<C: IOCollector> IOProfilingStats<C> {
     }
 
     fn track(&mut self, value: u64) {
-        let zend_thread = REQUEST_LOCALS.with(|cell| {
-            let locals = cell.borrow();
-            !locals.vm_interrupt_addr.is_null()
-        });
-        if !zend_thread {
+        if !REQUEST_LOCALS.with_borrow(|locals| !locals.vm_interrupt_addr.is_null()) {
```

Comment:
You may want to keep the variable for documentation purposes. !vm_interrupt_addr.is_null() does not convey that a "is zend_thread" check was meant.